data/BUILD.bazel

```diff
@@ -15,6 +15,7 @@ kt_android_library(
     deps = [
         ":dagger",
         "//model",
+        "//testing",
```

Comment:
This isn't correct--we shouldn't add test-only dependencies to a production target. This will leak test-only components in the production app. Instead, ``//testing`` should only be used for test targets.

The build bazel binary CI target would fail from adding this if not for the cycle failure being mentioned.

Reply:
Removed it.